### ISSUE TITLE
Milestone 05 Review

### ISSUE THREAD

**Issue by ps178** [mgood13/mgood13_Design_Skills_Project#9]
Please create issues for milestone reviews

Right now it looks like you only have the bottom half of the casing designed so I will wait for you to finish all of the CAD design before giving a review. One note I have is that dont forget the user would have to access the battery to change it. So, you have to think about how the top and bottom parts of your casing will attach together.

**Comment by mgood13** [comment]
The rest of the design has been added to the folder now. Sorry for the delay on it.
The hooks are intended to be an attachment mechanism between the two parts holding the lid onto the bottom.

**Comment by ps178** [comment]
CAD is looking good. Some notes:
-  There is only one opening (which is on the side). Is this for the button? How will the user see the LED?
- On the bottom part there are 4 cylinders on the surface. Is that the mount the arduino?

**Comment by mgood13** [comment]
You are completely right, forgot to add the hole for the button and also forgot to explain what I was intending to do... my bad.
The hole at the end is for the LED, the hole on the side is for the button, the structure right below that hole is to hold the button that I have.
The round posts in the middle will go under the PCB and the flag shaped posts are intended to hold the PCB in place.

**Comment by mgood13** [comment]
Also, I readied the corrected files to the Mechanical Designs folder
